lib/WebpackOptionsDefaulter.js

```diff
@@ -84,6 +84,10 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			{
 				test: /\.wasm$/i,
 				type: "webassembly/experimental"
+			},
+			{
+				test: /\.(png|jp?eg|svg)$/i,
```

Comment:
TBD

Reply:
If it helps, here are the regex used by some popular projects:
https://github.com/facebook/create-react-app/blob/b8da58499a8dd2e91feab117dedf127f0d9d59e4/packages/react-scripts/config/webpack.config.prod.js#L252
https://github.com/vuejs/vue-cli/blob/f8e97838dd25e37433ebbdb92b748c48b89f7c2a/packages/%40vue/cli-service/lib/config/base.js#L99
https://github.com/neutrinojs/neutrino/blob/7e5e3abb0bed6fca5f87679ff0a0ec0d276581d1/packages/image-loader/index.js#L10

Reply:
Personally in favor of defaults used by `neutrino`

Reply:
At any rate, there is a typo in the regexp; should be `jpe?g`